### PR TITLE
Generating claims SQL

### PR DESCRIPTION
**Pull request by dfaulken**
Will add more fields once Liza confirms how they match up. Would love a review of this mechanism of generating SQL.

Addresses #127.

### CODE REVIEW

File: app/models/incident.rb [hunk 1]
```diff
@@ -120,6 +120,34 @@ def reviewed?
...
   end
 
+  # from https://gist.github.com/janko-m/2b2cea3e8e21d9232fb9
+  def to_claims_sql
+    report = driver_incident_report
+    fields = {
+      street: report.location,
+      city: report.town,
+      state: 'MA',
+      # zip: driver.incident_report.zip,
```
Comment (Anbranin) [inline app/models/incident.rb]: Why is this commented out?

Comment (dfaulken) [reply app/models/incident.rb]: Field doesn't exist on this end yet.

Comment (Anbranin) [reply app/models/incident.rb]: o ok.

File: app/models/incident.rb [hunk 2]
```diff
@@ -120,6 +120,34 @@ def reviewed?
...
+      reason1: reason_code.identifier,
+      reason2: second_reason_code.try(:first, 3)
+    }
+
+    insert = Arel::Nodes::InsertStatement.new
+    insert.relation = Arel::Table.new(:incident)
+    insert.columns = fields.keys.map{ |field| insert.relation[field] }
+    insert.values = Arel::Nodes::Values.new(fields.values, insert.columns)
+
+    insert.to_sql
```
Comment (Anbranin) [inline app/models/incident.rb]: Damn. I tried this out in the console, I just wanted to see. it's pretty cool. Looks good.

File: app/models/incident.rb [hunk 3]
```diff
@@ -120,6 +120,34 @@ def reviewed?
     staff_reviews.present?
   end
 
+  # from https://gist.github.com/janko-m/2b2cea3e8e21d9232fb9
```
Comment (mboneil10) [inline app/models/incident.rb]: nice find

File: app/models/incident.rb [hunk 4]
```diff
@@ -120,6 +120,34 @@ def reviewed?
...
+      city: report.town,
+      state: 'MA',
+      # zip: driver.incident_report.zip,
+      longitude: longitude,
+      latitude: latitude,
+      'EmployeeID' => driver.badge_number,
+      'IncidentDesc' => root_cause_analysis,
+      'VehicleRouteNum' => report.block,
+      'VehicleNum' => report.bus,
+      'PointOfContact' => report.damage_to_bus_point_of_impact,
```
Comment (mboneil10) [inline app/models/incident.rb]: Unrelated to PR, why is this attribute so long when it could be called `PointOfContact`?

Comment (dfaulken) [reply app/models/incident.rb]: My feeling is that this attribute name is unambiguous. The impact could have occurred at a different point on the other vehicle, for example.

**Comment by dfaulken** [comment]
Thanks for reviewing everyone. Claims is MySQL as well, so hopefully no incompatibility there.

File: app/models/incident.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (dfaulken) [reply app/models/incident.rb]: Gotcha. Will do.

**Pull request closed without merging**